0-bootstrap/sa.tf

```diff
@@ -175,6 +175,31 @@ module "cicd_project_iam_member" {
   roles       = each.value
 }
 
+module "seed_project_remove_editor" {
+  source = "./modules/parent-iam-remove-role"
+
+  parent_type = "project"
+  parent_id   = module.seed_bootstrap.seed_project_id
```

Comment:
nit: Does for_each throw an error due to computed project_id?

Reply:
we can use a `for_each` if we use the map option.
Good Catch.